README.md

```diff
@@ -31,17 +31,47 @@ __gtk__ expects __GTK+__, __GLib__ and __Cairo__ development files to be install
 
 ### Windows
 
+Make sure you have the [GNU
+ABI](https://www.rust-lang.org/downloads.html#win-foot) version of the rust
+compiler installed.  Since we need our own toolchain make sure to uncheck
+"Linker and platform libraries" in the Rust setup.
+![Screenshot](rust_setup.png)
+
+If you're not sure, check if your Rust installation has `gcc.exe` and `ld.exe`
+in its `bin` directory, you may get a linking error `ld: cannot find -limm32`.
+In that case remove those executables, they will be provided by mingw instead.
+
+For the toolchain one has two possibilities:
+ 1. Install msys2 with a mingw toolchain and the gtk3 package
+ 2. Install the mingw-w64 toolchain separately and then install a GTK+ SDK
+
+#### 1. msys2 toolchain
+
+This method is recommended according to the [gtk project]
+(http://www.gtk.org/download/windows.php).  You can follow [this guide]
+(https://blogs.gnome.org/nacho/2014/08/01/how-to-build-your-gtk-application-on-windows/)
+or follow this steps:
+
+ - Install and update [msys2](https://msys2.github.io/)
+ - Install the mingw toolchain
+ ```
+ $ pacman -S mingw-w64-x86_64-toolchain
+ ```
+ - Install the gtk3 package
+ ```
+ $ pacman -S mingw-w64-x86_64-gtk3
+ ```
+
+#### 2. separate mingw toolchain
+
 Install [mingw-w64](http://mingw-w64.yaxm.org/) (select the win32 threading model) and download a __GTK+__ SDK:
- * The GNOME project has an official distribution of GTK+ 3.6: [x86](http://www.gtk.org/download/win32.php), [x64](http://www.gtk.org/download/win64.php).
+ * The GNOME project has [official distributions](http://win32builder.gnome.org/) of GTK+ 3.4 upto 3.10
```

Comment:
They're not official any more, let's just say they still host some binaries but recommend msys2.